spec/requests/users_spec.rb

```diff
@@ -6,32 +6,31 @@
       get '/users'
     end
     it 'returns http success' do
-      get '/users'
       expect(response).to have_http_status(:success)
     end
     it 'renders the correct template' do
       expect(response).to render_template('index')
     end
-    it 'has the correct text placeholderss' do
+    it 'has the correct text placeholders' do
       body = response.body
-      expect(body).to include('Users#index')
+      expect(body).to include('Users')
     end
   end
```

Comment:
- [ ] Hi @brahimdidi, let's together kindly revisit the requirements of this project. Based on the requirements of this project, we are supposed to write featured tests for the User index page that affirms the following:

- See the username of all other users.
- See the profile picture for each user.
- see the number of posts each user has written.
- Tests that redirect to the user's show page when you click on a user.

 Let's kindly fix the above issues.

Reply:
This is not this project tests, these are old tests for the requests that I fixed because some  reviewers just run rspec spec 
and when anything is falling they require changes on the project. THE TESTS ARE IN VIEW FOLDER PLEASE RUN rspec spec/views

Reply:
I did not add the include in the user's show page for comments  and posts because it only fetching the last 3 posts so it is only  3 queries it will never be n queries.